selfdrive/boardd/tests/test_boardd.cc

```diff
@@ -0,0 +1,87 @@
+#define CATCH_CONFIG_MAIN
+#include "catch2/catch.hpp"
+#include "cereal/messaging/messaging.h"
+#include "selfdrive/boardd/panda.h"
+
+const int CHUNK_SIZE = 64;
+std::string TEST_DATA = [](int size) {
```

Comment:
Ideally data length for each generated message should be any from dlc_to_len list.